webdriver/newsession/basic.py

```diff
@@ -0,0 +1,57 @@
+# META: timeout=long
```

Comment:
This should be under `tests\sessions`. Since we seem to be checking that the first and always match capabilities have sensible defaults, perhaps something like `default_values.py`?